pages/collection/[id]/play/index.js

```diff
@@ -0,0 +1,265 @@
+import PlayModeCard from "@/components/PlayModeCard";
+import styled from "styled-components";
+import { useState, useEffect } from "react";
+
+const StyledMain = styled.main`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: ${(props) => props.color || "#ffffff"};
+  flex-grow: 1;
+  min-height: 100vh;
+  :hover {
+    cursor: pointer;
+  }
+`;
+
+const StyledCountDown = styled.h2`
+  font-size: 7rem;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+`;
+
+const StyledHeader = styled.header`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 2.25rem;
+  position: fixed;
+  top: 2rem;
+  z-index: 100;
+  transition: transform 0.3s ease;
+`;
+
+const StyledHeadline = styled.h1`
+  display: flex;
+  justify-content: flex-start;
+`;
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  gap: 20px;
+`;
+
+const StyledButton = styled.button`
+  padding: 0.9rem;
+  border: 1.5px solid black;
+  min-width: 100px;
+  border-radius: 30px;
+  font-size: 1.3rem;
+  color: ${(props) => (props.stop ? "#fff" : "#000")};
+  background-color: ${(props) => (props.stop ? "#000" : "transparent")};
+
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+const StyledCardContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  width: 100%;
+  min-height: 100vh;
+  padding-bottom: 20px;
+`;
+
+const StyledMessageContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  padding-bottom: 20px;
+`;
+
+const StyledFooter = styled.footer`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  padding: 30px;
+  z-index: 200;
+`;
+
+export default function PlayMode({ collections, flashCards }) {
+  // Router
+  const router = useRouter();
+  const { id, card } = router.query; // id for remaining in current collection, card for moving to next card/page
+
+  // States
+  const [showStopConfirm, setShowStopConfirm] = useState(true);
+  const [showFinalMessage, setShowFinalMessage] = useState(false);
+  const [showAnswer, setShowAnswer] = useState(false);
+  const [isCorrect, setIsCorrect] = useState(0);
+  const [timeElapsed, setTimeElapsed] = useState(0);
+  const [countDown, setCountDown] = useState(3);
+  const [isCounting, setIsCounting] = useState(true);
+
+  // Countdown Timer: Runs until 0, then shows quiz content
+  useEffect(() => {
+    if (countDown > 0) {
+      const countDownTimer = setInterval(() => {
+        setCountDown((prevTime) => prevTime - 1);
+      }, 1000);
+
+      return () => clearInterval(countDownTimer);
```

Comment:
Instead of using an interval, please use a timeout for simplicity: https://stackblitz.com/edit/vitejs-vite-zjgle6ky?file=src%2FApp.jsx&terminal=dev